packages/Search/src/details/DTK_DetailsNode.hpp

```diff
@@ -22,6 +22,13 @@ struct Node
     KOKKOS_INLINE_FUNCTION
     Node() = default;
 
+    KOKKOS_INLINE_FUNCTION
+    Node( const Kokkos::pair<Node *, Node *> &c, const Box &bb )
```

Comment:
This constructor was needed to allow the list-initialization of the leaf nodes in `TreeConstruction:: initializeLeafNodes()`.